muted-tests.yml

```diff
@@ -332,8 +332,6 @@ tests:
 - class: org.elasticsearch.upgrades.VectorSearchIT
   method: testBBQVectorSearch {upgradedNodes=0}
   issue: https://github.com/elastic/elasticsearch/issues/121253
-- class: org.elasticsearch.lucene.FullClusterRestartLuceneIndexCompatibilityIT
-  issue: https://github.com/elastic/elasticsearch/issues/121257
```

Comment:
This is still muted on "main" but there's a [PR open to fix this test suite](https://github.com/elastic/elasticsearch/pull/121278)

Reply:
It's been merged now (but pending backport).